<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>40</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ClickableLabel" name="backBtt">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="searchBarRsc.qrc">:/png files/backBtt.png</pixmap>
   </property>
  </widget>
  <widget class="ClickableLabel" name="homeBtt">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="searchBarRsc.qrc">:/png files/homeBtt.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="searchBarRsc.qrc">:/png files/searchBarBG.png</pixmap>
   </property>
  </widget>
  <widget class="ClickableLabel" name="searchBtt">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="searchBarRsc.qrc">:/png files/searchBtt.png</pixmap>
   </property>
  </widget>
  <widget class="ClickableLabel" name="settingBtt">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Filipino" country="Philippines"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="searchBarRsc.qrc">:/png files/settingBtt.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nameLabel</string>
   </property>
  </widget>
  <widget class="SearchBox" name="searchBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>490</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::NoInsert</enum>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>backBtt</zorder>
  <zorder>homeBtt</zorder>
  <zorder>searchBtt</zorder>
  <zorder>settingBtt</zorder>
  <zorder>nameLabel</zorder>
  <zorder>searchBox</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.py</header>
  </customwidget>
  <customwidget>
   <class>SearchBox</class>
   <extends>QComboBox</extends>
   <header>SearchBox.py</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="searchBarRsc.qrc"/>
 </resources>
 <connections/>
</ui>
